repos/spack_repo/builtin/packages/kadath/package.py

```diff
@@ -20,14 +20,22 @@ class Kadath(CMakePackage):
     mixed spin binaries.
     """
 
-    homepage = "https://kadath.obspm.fr/fuka/"
-    git = "https://gitlab.obspm.fr/grandcle/Kadath.git"
+    homepage = "https://samueltootle.github.io/fuka/index.html"
+    git = "https://bitbucket.org/fukaws/fuka.git"
 
     maintainers("eschnett")
 
     license("GPL-3.0-or-later")
 
     version("fuka", branch="fuka")
+    version("2.3", tag="v2.3")
```

Comment:
I would recommend adding the commit of the tag as well unless the tag might change. This will speed up solver performance. 

https://spack.readthedocs.io/en/latest/spec_syntax.html#binary-provenance

Also outlined in the packaging guide https://spack.readthedocs.io/en/latest/packaging_guide_creation.html#git

Reply:
Done!